src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md

```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
-In this guide, learn how to create your own custom CrUX Dashboard to track an
-origin's user experience.
+big data sources, like the [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report/)
+(CrUX). In this guide, learn how to create your own custom CrUX Dashboard to
+track an origin's user experience trends.
 
-<img class="w-screenshot" src="./dash_fcp.png" alt="CrUX Dashboard">
+<img class="w-screenshot" src="./dash_lcp.png" alt="CrUX Dashboard">
 
 The CrUX Dashboard is built with a Data Studio feature called [Community
 Connectors](https://developers.google.com/datastudio/connector/).
 This connector is a pre-established link between the raw CrUX data on
-[BigQuery](https://console.cloud.google.com/bigquery?p=chrome-ux-report) and the visualizations of Data Studio. It eliminates
-the need for users of the dashboard to write any queries or generate any charts.
-Everything is built for you -- all you need is to provide an origin to look up.
-The connector code is also
-[open source](https://github.com/googledatastudio/community-connectors/tree/master/chrome-ux-report)
-if you want to see how it works.
+[BigQuery](https://console.cloud.google.com/bigquery?p=chrome-ux-report) 
+and the visualizations of Data Studio. It eliminates the need for users of the 
+dashboard to write any queries or generate any charts. Everything is built for 
+you; all you need is to provide an origin and a custom dashboard will be 
+generated for you.
 
 ## Creating a dashboard
 
 To get started, go to [g.co/chromeuxdash](https://g.co/chromeuxdash).
 This will take you to the CrUX community connector page where you can provide
-the origin for which the dashboard will be generated. Note that first-time users
-may need to complete permission or marketing preference prompts.
+the origin for which the dashboard will be generated. Note that first-time 
+users may need to complete permission or marketing preference prompts.
 
-<img class="w-screenshot" src="./dash_connector.png" alt="CrUX Data Studio connector">
+<img class="w-screenshot" src="./dash_connector.png" alt="CrUX Dashboard connector">
 
-The text input field accepts origins only, not full URLs. For example,
-`https://developers.google.com` is an origin but
-`https://developers.google.com/web/tools/chrome-user-experience-report/` is not.
-If you omit the protocol, HTTPS is assumed. Subdomains matter, so
-`developers.google.com` is very different from `google.com`.
+The text input field only accepts origins, not full URLs. For example:
+
+{% Compare 'better', 'Origin (Supported)' %}
+```text
+https://developers.google.com
+```
+{% endCompare %}
+
+{% Compare 'worse', 'URL (Not supported)' %}
+```text
+https://developers.google.com/web/tools/chrome-user-experience-report/
+```
+{% endCompare %}
+
+If you omit the protocol, HTTPS is assumed. Subdomains matter, for example
+`https://developers.google.com` and `https://www.google.com` are considered to 
+be very different origins.
```

Comment:
```suggestion
be different origins.
```
I don't think we want the emphasis of `very` here because it implies that there are shades/degrees of differentness in origins